ibm/operator_collection_sdk/playbooks/create_secret.yml

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright 2023 IBM Inc. All rights reserved
+# SPDX-License-Identifier: Apache2.0
+#
+
+---
+- name: Create secret
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  vars_prompt:
+    - name: operator_name
+      prompt: Enter the operator name to run the zoscb-encrypt tool for
+      private: false
+    - name: github_token
+      prompt: Enter the Github token if you are creating a github secret (Press Enter to create a regular secret)
+      private: true
+    - name: username
+      prompt: Enter you SSH Username for this endpoint (Press Enter to skip if creating a github secret)
+      private: false
+    - name: ssh_key
+      prompt: Enter the path to your private SSH Key for this endpoint (Press Enter to skip if creating a github secret)
+      private: false
+    - name: passphrase
+      prompt: Enter the passphrase for the SSH Key for this endpoint (Press Enter to skip if not using a passphrase or creating a github token)
+      private: true
+    - name: secret_name
+      prompt: Enter the name of the secret to create
+      private: false
+
+  tasks:
+
+    - name: Cluster setup
+      ansible.builtin.include_role:
+        name: cluster_setup
+
+    - name: Get SubOperator Pod Name
+      kubernetes.core.k8s_info:
+        kind: Pod
+        namespace: "{{ target_namespace }}"
+        label_selectors:
+          - "operator-name = {{ operator_name }}"
+      register: suboperator_pod_results
+
+    - name: Create github token secret
+      ansible.builtin.command: "oc exec -n {{ target_namespace }} {{ suboperator_pod_results.resources[0].metaname }} -- zoscb-encrypt github-token-secret -n github-token -f {{ github_token }}"
+      when: github_token != "" and ssh_key == ""
+
+    - name: Copy ssh key to suboperator pod
+      ansible.builtin.command: "oc cp {{ ssh_key }} {{ suboperator_pod_results.resources[0].metadata.name }}:/tmp/ssh_key"
+
+    - name: Create secret with passphrase
+      ansible.builtin.command: "oc exec -n {{ namespace }} {{ suboperator_pod_results.resources[0].metadata.name }} -- zoscb-encrypt credential-secret -n {{ secret_name }} -u {{ username }} -s {{ssh_key}} -p {{ passphrase }}"
+      when: passphrase != "" and github_token == ""
+
+    - name: Create secret without passphrase
+      ansible.builtin.command: "oc exec -n {{ target_namespace }} {{ suboperator_pod_results.resources[0].metadata.name }} -- zoscb-encrypt credential-secret -n {{ secret_name }} -u {{ username }} -s /tmp/key --namespace kberkos-dev"
```

Comment:
Since this pod contains multiple containers, you should also append the -c <container-name> parameter to the command